src/api/db/migrate/20211028111158_add_triggered_at_to_tokens.rb

```diff
@@ -0,0 +1,5 @@
+class AddTriggeredAtToTokens < ActiveRecord::Migration[6.1]
+  def change
+    add_column :tokens, :triggered_at, :datetime
```

Comment:
I wonder if it's more accurate naming the column as `last_triggering_at` instead of `triggered_at` :thinking:

Reply:
I would go with `last_trigger_at`.

Reply:
But doesn't `triggered_at` imply that it is the `last` trigger? Sure, I can rename it, but to me it aligns better with the terms `updated_at` and `created_at`

Reply:
I would go with `triggered_at`

Reply:
For me, `updated_at` implies "the last modification". With the `triggered_at` I don't have the same feeling.
Maybe because `updated_at` is too integrated into my brain. Anyway, I'm ok with `triggered_at`.